memiavl/snapshot.go

```diff
@@ -70,45 +83,38 @@ func OpenSnapshot(snapshotDir string) (*Snapshot, error) {
 	if format != SnapshotFormat {
 		return nil, fmt.Errorf("unknown snapshot format: %d", format)
```

Comment:
not sure how we migrate to new, we don't need replay whole if can support load from fallback snapshot right?

Reply:
let's worry about that after stablized, it's experimental for now.